<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionReclamationDialog</class> <widget class="QDialog" name="GestionReclamationDialog"> <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Claim Management</string> </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>List of Claims</string> </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableReclamations"> <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <property name="columnCount">
       <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>Claim ID</string> </property>
     </column>
     <column>
      <property name="text">
       <string>Client ID</string> </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string> </property>
     </column>
     <column>
      <property name="text">
       <string>Importance</string> </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string> </property>
     </column>
     <column>
      <property name="text">
       <string>Description (partial)</string> </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAjouter"> <property name="text">
        <string>Add Claim</string> </property>
       <property name="enabled">
        <bool>true</bool> </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModifier"> <property name="text">
        <string>Modify Claim</string> </property>
       <property name="enabled">
        <bool>false</bool> </property>
      </widget>
     </item>
     <item>
       <widget class="QPushButton" name="btnSupprimer"> <property name="text">
         <string>Delete Claim</string> </property>
        <property name="enabled">
         <bool>false</bool> </property>
       </widget>
      </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string> </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
